guides/introduction/quick_start_guide.md

```diff
@@ -114,3 +114,188 @@ This will lead you to a web page with a search bar that will allow you to search
+        }
+    })
+  </script>
+</html>
+```
+
+The code above comes in multiple parts:
+
+- In VueJs the DOM elements are written with a [declarative rendering](https://vuejs.org/v2/guide/#Declarative-Rendering) syntax. To make `instant-meilisearch` work with `Vue.js` some components must be added in the `<body>` tag. In the above example, `ais-instant-search`, `ais-search-box` and `ais-hits` are mandatory components to generate `instant-meilisearch` interface.
+- `<script src="..">` tags are [CDNs](https://en.wikipedia.org/wiki/Content_delivery_network) that import libraries needed to run `instant-meilisearch` with `Vue.js`.
+- The `Vue.js` part is where you customize your `instant-meilisearch`.
+
+To use `instant-meilisearch` in `Vue.js` using `npm` or `yarn` please visit [meilisearch-vue](https://github.com/meilisearch/meilisearch-vue).
+
+:::
+
+::: tab React
+
+The following code sample uses [React](https://reactjs.org/) framework.
+
+```html
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/templates/basic_search.css" />
+</head>
+<html>
+  <body>
+      <div id="app" class="wrapper"></div>
+  </body>
+  <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+  <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+  <script src="https://cdn.jsdelivr.net/npm/react-instantsearch-dom@6.7.0/dist/umd/ReactInstantSearchDOM.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js"></script>
+  <script>
+    const { InstantSearch, SearchBox, Hits, Highlight, Configure }  = ReactInstantSearchDOM;
+    const searchClient = instantMeiliSearch(
+      "http://localhost:7700"
+    );
+
+    const App = () => (
+      React.createElement(InstantSearch, {
+        indexName: "movies",
+        searchClient: searchClient
+      }, [
+        React.createElement(SearchBox, { key: 1 }),
+        React.createElement(Hits, { hitComponent: Hit, key: 2 }),
+        React.createElement(Configure, { hitsPerPage: 10 })]
+      )
+    );
+    function Hit(props) {
+        return React.createElement(Highlight, {
+          attribute: "title",
+          hit: props.hit
+        })
+    }
+    const domContainer = document.querySelector('#app');
+    ReactDOM.render(React.createElement(App), domContainer);
+  </script>
+</html>
+```
+
+The code above comes in multiple parts:
+
+- The `<body>` of the page is the entry point for `react`.
+  `instant-meilisearch` adds the search bar and search results in this entry point.
+- `<script src="..">` tags are [CDNs](https://en.wikipedia.org/wiki/Content_delivery_network) that import libraries needed to run `instant-meilisearch` in [React](https://reactjs.org/).
+- The React script (last `<script>` tag in the file) is where you customize your `instant-meilisearch`.
```

Comment:
```suggestion
- The React `<script>` at the end is where you can customize `instant-meilisearch`.
```

Reply:
This is not completely accurate. As it is not a `React <script>` But a <script> tag containing a script using React.

I will nontheless make a change to make this less confusing :)